### PR TITLE
Fixesss

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Fixes round-start air alarms. The server room now won't alert even when it's 0K, and the labor camp no longer has random tiles that are 0C.
2) Adds reference collection to the garbage collector. By uncommenting the TESTING definition in the compile, you'll have access to additional debugging tools to help you find references to objects that are being qdel'd. See https://github.com/tgstation/-tg-station/pull/11458 for more information. (By duncathan).
3) Fixes the syringe gun right in-hand. For some reason it was animated, and had the burning flag as a few frames.
4) Fixes being unable to pick up tile stacks due to a missed space in the tile file (oh, BYOND).
5) Fixes Wizards being able to gain INFINITE spells by upgrading and then refunding their spells due to it not being removed properly.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Just a word of warning towards anyone keen on testing out checking references: don't do it unless you know what you're doing.

Checking for references can be very taxing on the server as it searches for everything that references it; only use this if you really and truly have intentions of fixing qdel/GC related issues yourself or fully understand what it is that you're checking for.

**Comment by Markolie** [comment]
You can't actually do it on the live server - it's only possible if you define TESTING in the compile options, which by default isn't defined. It should never be defined in this Github either - only locally.

**Comment by Fox-McCloud** [comment]
Alright, good, because, well..yeah, it can be quiteee expensive.

Also, I thought we already had a way to check references? Hmm..

**Comment by Markolie** [comment]
Not sure - I certainly couldn't find it. Plus, this is a very robust way of doing it (it's possible by both setting the qdel hint and simply through a verb).

**Comment by Fox-McCloud** [comment]
It's righttttt here: https://github.com/ParadiseSS13/Paradise/blob/1f7bfe92bea16b2a9c6ed5557491e184ea3b5244/code/modules/garbage%20collection/gc_testing.dm#L127

**Comment by Fox-McCloud** [comment]
Will be nice to finally find some references to datums...provided that some datums don't actually have many references, we can start GCing some of them, as well; once the reagents fixup is in, that should ensure that they GC *very very* well with very few exceptions.

**Comment by Markolie** [comment]
I'll just get rid of the old find_references proc - it's basically the old one /tg/ used before switching to this one. After that this can safely be merged.

**Comment by Fox-McCloud** [comment]
All the yes.

GC all the things.

File: code/modules/garbage collection/garbage_collector.dm [hunk 1]
```diff
@@ -176,3 +181,72 @@ var/global/datum/controller/process/garbage_collector/garbageCollector
 
 /proc/gcwarning(msg)
 	log_to_dd("## GC WARNING: [msg]")
+	
+#ifdef TESTING
```
Comment (tigercat2000) [inline code/modules/garbage collection/garbage_collector.dm]: all of this should really be in gc_testing.dm

**Pull request merged**